shared/src/business/entities/Batch.js

```diff
@@ -31,7 +31,7 @@ Batch.validationName = 'Batch';
  *
  * @param {object} page the page to add
  * @returns {Batch} the batch entity after the page is added
-
+ 
```

Comment:
very minor, but it doesn't seem like this is desired

Reply:
Would you like for us to push a change for this, or could it come through on the next PR?

Reply:
next PR is totally fine